src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+/**
+ * Feel free to remove this class and create your own.
+ */
+public class Main {
+    public static void main(String[] args) {
+        Figure[] figures = new Figure[6];
+        FigureSupplier figureSupplier = new FigureSupplier();
+
+        for (int i = 0; i < figures.length; i++) {
+            figures[i] = (i <= figures.length / 2 - 1)
+                    ? figureSupplier.getRandomFigure()
+                    : figureSupplier.getDefaultFigure();
+        }
```

Comment:
You can calculate the middle of figures array before loop and just use it value in if statement